src/functions.cpp:1693:29: warning: unused variable 'out'

c++ -g -Wall -DLIBSASS_VERSION="\"3.3.3-19-g3054-dirty\"" -std=c++0x -I /home/saper/sw/libsass/include -fPIC -fPIC -c -o src/functions.o src/functions.cpp
src/functions.cpp:1693:29: warning: unused variable 'out' [-Wunused-variable]
        Sass_Output_Options out(ctx.c_options);
                            ^
1 warning generated.


<bountysource-plugin>
---
Want to back this issue? **[Post a bounty on it!](https://www.bountysource.com/issues/30846548-src-functions-cpp-1693-29-warning-unused-variable-out?utm_campaign=plugin&utm_content=tracker%2F283068&utm_medium=issues&utm_source=github)** We accept bounties via [Bountysource](https://www.bountysource.com/?utm_campaign=plugin&utm_content=tracker%2F283068&utm_medium=issues&utm_source=github).
</bountysource-plugin>

--- a/src/functions.cpp
+++ b/src/functions.cpp
@@ -1690,7 +1690,6 @@ namespace Sass {
         Sass_Output_Style old_style;
         old_style = ctx.c_options.output_style;
         ctx.c_options.output_style = TO_SASS;
-        Sass_Output_Options out(ctx.c_options);
         Emitter emitter(ctx.c_options);
         Inspect i(emitter);
         i.in_declaration = false;

